File: modules/wem-web/src/main/java/com/enonic/cms/web/rest/userstore/UserStoreConnectorModel.java
package com.enonic.cms.web.rest.userstore;

public final class UserStoreConnectorModel
{

    private String name;

    private String pluginType;

    private boolean canCreateUser;

    private boolean canUpdateUser;

    private boolean canUpdateUserPassword;

    private boolean canDeleteUser;

    private boolean canCreateGroup;

    private boolean canUpdateGroup;

    private boolean canReadGroup;

    private boolean canDeleteGroup;

    private boolean groupsLocal;


    public String getName()
    {
        return name;
    }

    public void setName( String name )
    {
        this.name = name;
    }

    public String getPluginType()
    {
        return pluginType;
    }

    public void setPluginType( String pluginType )
    {
        this.pluginType = pluginType;
    }

    public boolean isCanCreateUser()
    {
        return canCreateUser;
    }

    public void setCanCreateUser( boolean canCreateUser )
    {
        this.canCreateUser = canCreateUser;
    }

    public boolean isCanUpdateUser()
    {
        return canUpdateUser;
    }

    public void setCanUpdateUser( boolean canUpdateUser )
    {
        this.canUpdateUser = canUpdateUser;
    }

    public boolean isCanUpdateUserPassword()
    {
        return canUpdateUserPassword;
    }

    public void setCanUpdateUserPassword( boolean canUpdateUserPassword )
    {
        this.canUpdateUserPassword = canUpdateUserPassword;
    }

    public boolean isCanDeleteUser()
    {
        return canDeleteUser;
    }

    public void setCanDeleteUser( boolean canDeleteUser )
    {
        this.canDeleteUser = canDeleteUser;
    }

    public boolean isCanCreateGroup()
    {
        return canCreateGroup;
    }

    public void setCanCreateGroup( boolean canCreateGroup )
    {
        this.canCreateGroup = canCreateGroup;
    }

    public boolean isCanUpdateGroup()
    {
        return canUpdateGroup;
    }

    public void setCanUpdateGroup( boolean canUpdateGroup )
    {
        this.canUpdateGroup = canUpdateGroup;
    }

    public boolean isCanReadGroup()
    {
        return canReadGroup;
    }

    public void setCanReadGroup( boolean canReadGroup )
    {
        this.canReadGroup = canReadGroup;
    }

    public boolean isCanDeleteGroup()
    {
        return canDeleteGroup;
    }

    public void setCanDeleteGroup( boolean canDeleteGroup )
    {
        this.canDeleteGroup = canDeleteGroup;
    }

    public boolean isGroupsLocal()
    {
        return groupsLocal;
    }

    public void setGroupsLocal( boolean groupsLocal )
    {
        this.groupsLocal = groupsLocal;
    }
}

File: modules/wem-webapp/src/main/webapp/admin/resources/app/model/userstore/UserstoreConnectorModel.js
Ext.define('Admin.model.userstore.UserstoreConnectorModel', {
    extend: 'Ext.data.Model',

    idProperty: 'name',

    fields: [
        'name', 'pluginType',
        {name: 'canCreateUser', type: 'boolean', defaultValue: false },
        {name: 'canUpdateUser', type: 'boolean', defaultValue: false },
        {name: 'canUpdateUserPassword', type: 'boolean', defaultValue: false },
        {name: 'canDeleteUser', type: 'boolean', defaultValue: false },
        {name: 'canCreateGroup', type: 'boolean', defaultValue: false },
        {name: 'canUpdateGroup', type: 'boolean', defaultValue: false },
        {name: 'canReadGroup', type: 'boolean', defaultValue: false },
        {name: 'canDeleteGroup', type: 'boolean', defaultValue: false },
        {name: 'canResurrectDeletedGroups', type: 'boolean', defaultValue: false },
        {name: 'canResurrectDeletedUsers', type: 'boolean', defaultValue: false },
        {name: 'groupsLocal', type: 'boolean', defaultValue: true }
    ]
});
